<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewer</class>
 <widget class="QMainWindow" name="ImageViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>361</width>
      <height>211</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>359</width>
       <height>209</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_pic">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>3</y>
        <width>281</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>386</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Open">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_as"/>
    <addaction name="action_Print"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="action_Cut"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Zoom_in"/>
    <addaction name="action_Zoom_out"/>
    <addaction name="action_Zoom_100"/>
    <addaction name="action_Fit_to_Window"/>
    <addaction name="action_Show_Dialog"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_About_Qt"/>
   </widget>
   <addaction name="menu_Open"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="action_Save_as"/>
   <addaction name="action_Print"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Paste"/>
   <addaction name="action_Zoom_in"/>
   <addaction name="action_Zoom_out"/>
  </widget>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/images/open.png</normaloff>:/images/images/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Save_as">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/images/save_as.png</normaloff>:/images/images/save_as.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save &amp;as</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/images/print.png</normaloff>:/images/images/print.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/images/exit.png</normaloff>:/images/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/images/copy.png</normaloff>:/images/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/images/paste.png</normaloff>:/images/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Zoom_in">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/images/zoom-png-1.png</normaloff>:/images/images/zoom-png-1.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+]</string>
   </property>
  </action>
  <action name="action_Zoom_out">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/images/zoom-out.png</normaloff>:/images/images/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="action_Zoom_100">
   <property name="text">
    <string>&amp;Zoom 100%</string>
   </property>
  </action>
  <action name="action_Fit_to_Window">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Fit to Window</string>
   </property>
  </action>
  <action name="action_Show_Dialog">
   <property name="text">
    <string>&amp;Show Dialog</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/images/info.png</normaloff>:/images/images/info.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_About_Qt">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/images/info.png</normaloff>:/images/images/info.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About Qt</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/images/new.png</normaloff>:/images/images/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/images/images/cut.png</normaloff>:/images/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
